alpha/apps/kaltura/modules/extwidget/actions/serveFlavorAction.class.php

```diff
@@ -186,14 +207,16 @@ protected function servePlaylist($entry, $captionLanguages)
 		}
 		
 		list($entryIds, $durations, $referenceEntry, $captionFiles) = myPlaylistUtils::executeStitchedPlaylist($entry, $captionLanguages);
+		list($offsets, $durations) = $this->retrievePlaylistAttributesData($entry, $durations);
 		$this->serveEntriesAsPlaylist($entryIds, $durations, $referenceEntry, $entry, null,
-			$captionFiles, $captionLanguages, $isLive, 0, 0, 0, 0);
+			$captionFiles, $captionLanguages, $isLive, 0, 0, 0, 0, $offsets);
 	}
 
 	protected function serveEntriesAsPlaylist($entryIds, $durations, $referenceEntry, $origEntry, $flavorParamIds,
 	                                          $captionFiles, $captionLanguages, $isLive,
-	                                          $playlistStartTime, $firstClipStartTime, $initialClipIndex, $initialSegmentIndex)
+	                                          $playlistStartTime, $firstClipStartTime, $initialClipIndex, $initialSegmentIndex, $offsets=null)
 	{
+		$offsets = $offsets ?? array_fill(0, count($entryIds), 0);
```

Comment:
?? works only from php 7, there are on prems will older php version so please write the full condition